src/rebar_prv_common_test.erl

```diff
@@ -704,7 +705,7 @@ handle_keep_logs(LogDir, N) ->
                     SortedDirs = lists:reverse(lists:sort(Dirs)),
                     %% sort the log dirs and keep the N - 1 newest
                     {_Keep, Discard} = lists:split(N - 1, SortedDirs),
-                    ?DEBUG("Removing the following directories because keep_logs option was found: ~p", [Discard]),
+                    ?DEBUG("Removing the following directories because keep_logs is in ct_opts: ~p", [Discard]),
```

Comment:
Confused by this, its changing from the option was "not found" to saying it was found?

Reply:
The `{keep_logs, N}` option keeps only the last `N` runs of CT data. That way if your test suites are leaving a large footprint you can regulate the size on disk by only keeping the last 5 runs rather than ending up with 150 of them.

The way the output looks now is:

```
===> Removing the following directories because keep_logs is in ct_opts: ["ct_run.nonode@nohost.2020-10-13_13.57.34"]
===> Running tests with {ct_opts, [{auto_compile,false},
                                          {logdir,
                                           "/home/ferd/code/self/rebar3/_build/test/logs"},
                                          {abort_if_missing_suites,true},
                                          {ct_hooks,
                                           [cth_readable_failonly,
                                            cth_readable_compact_shell,
                                            cth_retry]},
                                          {keep_logs,5},
                                          {suite,
                                           ["/home/ferd/code/self/rebar3/_build/test/lib/rebar/test/rebar_as_SUITE.erl"]}]}.
```
The message is only shown when the log line is deleted and the `ct_opts` field is right below it showing `{keep_logs, 5}` being set in our config file.